test/classes/diten/multipledestructor.chpl

```diff
@@ -1,7 +1,7 @@
 class C {
-  proc ~C() { writeln("In ~C #1"); }
+  proc deinit() { writeln("In deinit #1"); }
 }
-proc C.~C() { writeln("In ~C #2"); }
+proc C.deinit() { writeln("In deinit #2"); }
```

Comment:
Should we update the name of this test, to reflect that it exercises deinitializers now instead of destructors?